.github/workflows/visual-regression.yml

```diff
@@ -1,9 +1,8 @@
 name: Visual regression tests with Netlify
 
 on:
-  pull_request:
-    paths:
-      - 'optuna/visualization/**'
+  pull_request_target:
+    types: [labeled]
```

Comment:
[Notes] The limitation of the path was removed, but it sounds reasonable to me since the workflows are manually triggered.

Reply:
[Notes] The document of `pull_request_target` is [this one](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target). I think the change follows the usage instruction.

Reply:
Thank you for the additional clarifications.